packages/mephisto-worker-experience/src/Tips.js

```diff
@@ -0,0 +1,188 @@
+import React, { useState, Fragment, useReducer } from "react";
+import { usePopperTooltip } from "react-popper-tooltip";
+import { handleTipSubmit } from "./Functions";
+import { useMephistoTask } from "mephisto-task";
+import "./index.css";
+import "react-popper-tooltip/dist/styles.css";
+import { tipsReducer } from "./Reducers";
+import InfoIcon from "./InfoIcon";
+
+function Tips({
+  list,
+  handleSubmit,
+  disableUserSubmission,
+  headless,
+  maxHeight,
+  maxWidth,
+  placement,
+}) {
+  const [isVisible, setIsVisible] = useState(false);
+  const [state, dispatch] = useReducer(tipsReducer, {
+    status: 0,
+    text: "",
+  });
+
+  const maxPopupHeight = maxHeight ? maxHeight : "30rem";
+  const maxPopupWidth = maxWidth ? maxWidth : "30rem";
+  const {
+    getTooltipProps,
+    setTooltipRef,
+    setTriggerRef,
+    visible,
+  } = usePopperTooltip(
+    {
+      trigger: "click",
+      closeOnOutsideClick: true,
+      visible: isVisible,
+      offset: [0, 6],
+      onVisibleChange: setIsVisible,
+    },
+    {
+      placement: placement ? placement : "top-start",
+    }
+  );
+  const { taskConfig, handleMetadataSubmit } = useMephistoTask();
+  const tipsArr = (list ? list : []).concat(
+    taskConfig ? taskConfig["metadata"]["tips"] : []
+  );
+  const headlessPrefix = headless ? "headless-" : "";
+  const [tipData, setTipData] = useState({ header: "", text: "" });
+  const tipsComponents = tipsArr.map((tip, index) => {
+    return (
+      <li
+        key={`tip-${index + 1}`}
+        className={`${headlessPrefix}mephisto-worker-experience-tips__tip`}
```

Comment:
I would suggest you consider pulling the `mephisto-worker-experience` portion also into a prefix var,maybe just call it `stylePrefix` or something, so that:

```jsx
const headlessPrefix = headless ? "headless-" : "";
const stylePrefix = `${headlessPrefix}mephisto-worker-experience-tips__`

<div className={`${stylePrefix}tip`} />

// or:

const s = className => `${headlessPrefix}mephisto-worker-experience-tips__${className}`

<div className={s('tip')} />
```

makes the classnames in the html more readable, less verbose/repetitive

A less important reason is that that now that we'll have to rename `mephisto-worker-experience` to `mephisto-worker-addons` it's easier to make changes like that. Although this is not as important bc we won't be changing this name again, and with find/replace it's pretty easy to fix.